integration_tests/mocked_data.py

```diff
@@ -21,22 +16,22 @@ class MockedSample(NamedMock):
     real_name='artpool',
     input_artifact_list=Mock(
         return_value=[
-            NamedMock(reagent_labels=['D701-D502 (ATTACTCG-ATAGAGGC)'],
-                      samples=[MockedSample(real_name='10015AT0001', id='LP6002014-DTP_A01')]),
-            NamedMock(reagent_labels=['D702-D502 (TCCGGAGA-ATAGAGGC)'],
-                      samples=[MockedSample(real_name='10015AT0002', id='LP6002014-DTP_A02')]),
-            NamedMock(reagent_labels=['D703-D502 (CGCTCATT-ATAGAGGC)'],
-                      samples=[MockedSample(real_name='10015AT0003', id='LP6002014-DTP_A03')]),
-            NamedMock(reagent_labels=['D704-D502 (GAGATTCC-ATAGAGGC)'],
-                      samples=[MockedSample(real_name='10015AT0004', id='LP6002014-DTP_A04')]),
-            NamedMock(reagent_labels=['D705-D502 (ATTCAGAA-ATAGAGGC)'],
-                      samples=[MockedSample(real_name='10015AT0006', id='LP6002014-DTP_A05')]),
-            NamedMock(reagent_labels=['D706-D502 (GAATTCGT-ATAGAGGC)'],
-                      samples=[MockedSample(real_name='10015AT0007', id='LP6002014-DTP_A06')]),
-            NamedMock(reagent_labels=['D707-D502 (CTGAAGCT-ATAGAGGC)'],
-                      samples=[MockedSample(real_name='10015AT0008', id='LP6002014-DTP_A07')]),
-            NamedMock(reagent_labels=['D708-D502 (TAATGCGC-ATAGAGGC)'],
-                      samples=[MockedSample(real_name='10015AT0009', id='LP6002014-DTP_A08')])
+            Mock(reagent_labels=['D701-D502 (ATTACTCG-ATAGAGGC)'],
```

Comment:
What's the difference between NamedMock and Mock?

Reply:
NamedMock defines a property that works around `unittest.Mock` behaving weirdly if it's mocking an object that has an attribute called `name` - see tests/test_analysisdriver.py. These top level objects don't need to be NamedMocks